app/javascript/chat/ChatChannelSettings/ChatChannelSettingsSection.jsx.erb

```diff
@@ -74,7 +74,7 @@ const ChatChannelSettingsSection = ({
     />
     <ModFaqSection
       currentMembershipRole={currentMembership.role}
-      email="yo@dev.to"
+      email="<%= email_link %>"
```

Comment:
I thought about this and instead of placing the ERB template here, make the pack file that loads chat the ERB template, as pack files are typically not tested. It can be passed in as a prop to the `<Chat />` component like this. This will remove the need to transform ERB files in jest and will also make testing easier as it's just a prop.

```jsx
import { h, render } from 'preact';
import Chat from '../chat/chat';
import { Snackbar } from '../Snackbar/Snackbar';

function loadElement() {
  const root = document.getElementById('chat');

  if (root) {
    render(<Snackbar lifespan="3" />, document.getElementById('snack-zone'));
    render(<Chat {...root.dataset} modFaqEmail="<%= email_link %>" />, root);

    const placeholder = document.getElementById('chat_placeholder');

    if (placeholder) {
      root.removeChild(placeholder);
    }
  }
}

window.InstantClick.on('change', () => {
  loadElement();
});

loadElement();

```

Reply:
> I thought about this and instead of placing the ERB template here, make the pack file that loads chat the ERB template, as pack files are typically not tested. It can be passed in as a prop to the `<Chat />` component like this. This will remove the need to transform ERB files in jest and will also make testing easier as it's just a prop.
> 
> ```js-jsx
> import { h, render } from 'preact';
> import Chat from '../chat/chat';
> import { Snackbar } from '../Snackbar/Snackbar';
> 
> function loadElement() {
>   const root = document.getElementById('chat');
> 
>   if (root) {
>     render(<Snackbar lifespan="3" />, document.getElementById('snack-zone'));
>     render(<Chat {...root.dataset} modFaqEmail="<%= email_link %>" />, root);
> 
>     const placeholder = document.getElementById('chat_placeholder');
> 
>     if (placeholder) {
>       root.removeChild(placeholder);
>     }
>   }
> }
> 
> window.InstantClick.on('change', () => {
>   loadElement();
> });
> 
> loadElement();
> ```

This is much better solution and it also removes lot other issues that might arise when mixing jsx and erb.